extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/VertxHybridPoolObjectMapperCustomizer.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.jackson.deployment;
```

Comment:
This actually needs to be in the `runtime` module of the extension

Reply:
Fixed.

Reply:
Excellent, thanks